resources/sdk/pureclouddotnet/templates/ApiClient.mustache

```diff
@@ -211,7 +239,13 @@ namespace {{packageName}}.Client
             var request = PrepareRequest(
                 path, method, queryParams, postBody, headerParams, formParams, fileParams,
                 pathParams, contentType);
-            var response = await RestClient.ExecuteTaskAsync(request);
+
+             Retry retry = new Retry(this.RetryConfig);
+             IRestResponse response;
+            do{
```

Comment:
fix indentation

Reply:
fixed

Reply:
It's still wonky
![image](https://user-images.githubusercontent.com/1349974/78177930-1ea11180-741c-11ea-962f-66a09b34997f.png)

Reply:
Sorry, I missed that before. fixed now.